### PR TITLE
Ta debug

### LINKED ISSUE #25: Remove "jerky" acceleration
## Summary
- The acceleration is not constant, but it's "pumping" according to the reviewer
- Still needs to investigate what causing it

### PR DESCRIPTION
**Pull request by Krishtof-Korda**
## Summary

Fix #25

### CODE REVIEW

**Review by kkweon — CHANGES_REQUESTED** [review]
:+1: Let's do it.  First, let's revert the `tl_detector.py` first. It's using the ground truth right now.

File: ros/src/tl_detector/tl_detector.py [hunk 1]
```diff
@@ -36,9 +36,9 @@ def __init__(self):
         self.last_state = TrafficLight.UNKNOWN
         self.last_wp = -1
         self.state_count = 0
-        self.publish_ground_truth = False
+        self.publish_ground_truth = True
```
Comment (kkweon) [inline ros/src/tl_detector/tl_detector.py]: So, this should be `false` :thinking:

File: ros/src/tl_detector/tl_detector.py [hunk 2]
```diff
@@ -36,9 +36,9 @@ def __init__(self):
...
         self.has_image = False
-        self.enable_early_stop = True
+        self.enable_early_stop = False
```
Comment (kkweon) [inline ros/src/tl_detector/tl_detector.py]: Not sure of this one

**Review by kkweon — CHANGES_REQUESTED** [review]

File: ros/src/tl_detector/tl_detector.py (hunk 1 above)
Comment (Krishtof-Korda) [reply ros/src/tl_detector/tl_detector.py]: correct 👍

File: ros/src/tl_detector/tl_detector.py (hunk 2 above)
Comment (Krishtof-Korda) [reply ros/src/tl_detector/tl_detector.py]: This one will be True.

**Review by kkweon — APPROVED** [review]
Ran a test. It looks good to me.

**Pull request merged**